### PR TITLE
Update configuration File (php.ini) Path

### PR DESCRIPTION
**Pull request by shafan**
The configuration File Path is not /usr/local/bin but /usr/local/etc/php

### CODE REVIEW

**Comment by tianon** [comment]
LGTM:
```
PHP_INI_DIR => /usr/local/etc/php
```

**Comment by yosifkit** [comment]
LGTM

**Comment by yosifkit** [comment]
Can you put this in the content.md?

**Comment by shafan** [comment]
ok

**Comment by yosifkit** [comment]
Thanks!

**Comment by shafan** [comment]
Now it's good

File: php/content.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ We recommend that you add a custom `php.ini` configuration. `COPY` it into
 same commands to build and run:
```
Comment (tianon) [inline php/content.md]: There's one more in this paragraph that you appear to have missed when you copied the change over. :+1:

**Comment by shafan** [comment]
oups sorry, now it's good

**Comment by tianon** [comment]
LGTM

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**